django_x509/base/admin.py

```diff
@@ -59,11 +60,25 @@ class CaAdmin(BaseAdmin):
     list_filter = ['key_length', 'digest', 'created']
 
 
+class BaseX509Form(forms.ModelForm):
+        OPERATION_CHOICES = (
+            ('', ''),
```

Comment:
Should change the display text from '' to something more intuitive.

Reply:
'----' is the default value in other fields, so I'll change it to that